scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -38,19 +32,17 @@
         "clusterResourceGroup": "[split(parameters('aksResourceId'),'/')[4]]",
         "clusterName": "[split(parameters('aksResourceId'),'/')[8]]",
         "clusterLocation": "[replace(parameters('aksResourceLocation'),' ', '')]",
-        "workspaceSubscriptionId": "[split(parameters('workspaceResourceId'),'/')[2]]",
-        "workspaceResourceGroup": "[split(parameters('workspaceResourceId'),'/')[4]]",
         "dcrName": "[Concat('MSCI', '-', variables('clusterName'), '-', variables('clusterLocation'))]",
         "associationName":  "ContainerInsightsExtension",
-        "dataCollectionRuleId": "[resourceId(variables('workspaceSubscriptionId'), variables('workspaceResourceGroup'), 'Microsoft.Insights/dataCollectionRules', variables('dcrName'))]"
+        "dataCollectionRuleId": "[resourceId(variables('clusterSubscriptionId'), variables('clusterResourceGroup'), 'Microsoft.Insights/dataCollectionRules', variables('dcrName'))]"
```

Comment:
what's the reason for moving DCR resource to cluster RG instead of workspace RG? The reason for keeping the DCR in workspace RG to avoid data loss if the cluster RG gets deleted.  Did you check moving DCR to cluster RG doesnt have any impact?

Reply:
Two reasons:

- We're creating the prometheus DCR's in the cluster's resource group and wanted to keep consistency between what both logs and CI are doing.
- The naming convention we're following only contains the cluster name and cluster location. This was causing an issue if we're using another cluster with the same and same location but sending data to the same workspace (same issue exists on the prom metrics side) as there would be conflicting DCRs. Hence the logic was to move the DCR into the clusters resource group so that we can avoid the conflict.

Yes, currently the UX private preview flight for the MAC, prometheus changes is also creating the DCR in the cluster RG and data is flowing as expected.

Reply:
Did you check with AMCS team whether they will be any issue accessing the historical data if the DCR Resource gets deleted (which is common if the customer deletes the cluster RG)?

Reply:
Discussed with AMCS team and they confirmed there is no impact accessing historical data when the DCR deleted.